admin/class-admin-init.php

```diff
@@ -130,6 +131,86 @@ private function get_yoast_seo_suggested_plugins_notification( $name, $plugin, $
 		);
 	}
 
+	/**
+	 * Determines whether a update notification needs to be displayed.
+	 *
+	 * @return void
+	 */
+	public function yoast_plugin_update_notification() {
+		$notification_center = Yoast_Notification_Center::get();
+		$current_version     = WPSEO_Options::get( 'version', WPSEO_VERSION );
+
+		$file = plugin_dir_path( WPSEO_FILE ) . 'release-info.json';
+		$release_json = file_get_contents( $file );
+		/**
+		 * Filter: 'wpseo_update_notice_content' - Allow filtering of the content
+		 * of the update notice read from the release-info.json file.
+		 *
+		 * @api object The object from the release-info.json file.
+		 */
+		$release_info = apply_filters( 'wpseo_update_notice_content', json_decode( $release_json ) );
+
+		// Remove if file is not present, malformed or for a different version.
+		if ( is_null( $release_info )
+			|| empty( $release_info->version )
+			|| (float) $release_info->version !== (float) $current_version
+		 	|| empty( $release_info->release_description )
+		) {
+			$notification_center->remove_notification_by_id( 'wpseo-plugin-updated' );
+			return;
+		}
+
+		$notification  = $this->get_yoast_seo_update_notification( $release_info );
+		$dismissal_key = $notification->get_dismissal_key();
```

Comment:
I'd suggest just calling `$notification->get_dismissal_key()` directly in the `get_user_option` as this variable isn't used anywhere else.